### PR TITLE
Added a S206 vector. Work in progress.

### PR DESCRIPTION
**Pull request by MattisLind**
PR for the S206 file so that you can have a look.

### CODE REVIEW

File: vectors/s206.fct [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Test for S206, Dual Flip-Flop
+
```
Comment (anders-bzn) [inline vectors/s206.fct:2]: A a comment line with "Work in progress, not fully verified"

File: vectors/s206.fct [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+# Outputs are P and R
+# General clear input on pin F
+
+
+# -3V = 1
+# 0V = 0
+
+# Define the limits when testing inputs
+input-current='-1.0mA'
+input-current-margin='20.0mA'
```
Comment (anders-bzn) [inline vectors/s206.fct:21]: Add:
# High margin for testing,

File: vectors/s206.fct [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+input-active-level='high'
+input-logic-high='-300mV'
```
Comment (anders-bzn) [inline vectors/s206.fct:23]: I thing this value are a bit to low. Use -50mV.
The value is being substrated and the differens should not be more that +/- 100mV. So 300mV will fail the input tests.

**Review by anders-bzn — COMMENTED** [review]
Very nice with a lot of comments. I have just minor things. 

Please add the pdf of the schematics used for creating the vector to the drawings folder in the same commit.

File: vectors/s206.fct (hunk 2 above)
Comment (MattisLind) [reply vectors/s206.fct:21]: I reverted back to the original 5.0 mA margin since it didn't make any difference.

**Comment by MattisLind** [comment]
Ok, perfect. I add the doc files from logic handbook and the schematic. You can close this PR and not merge it. I will come up with a new PR later on.

**Comment by MattisLind** [comment]
Added docs now and fixed your comments

**Pull request merged**